charts/cryostat/values.yaml

```diff
@@ -198,6 +198,16 @@ authentication:
     ## @param authentication.basicAuth.filename Key within Secret containing the properties file. The properties file should contain one user per line, with the syntax "user=passHex", where "user" is the username and "passHex" is the SHA-256 hash of the desired password **(Required if basicAuth is enabled)**
     filename: ""
 
+oauth2Proxy:
+  ## @param authentication.oauth2Proxy.enabled Whether Cryostat should use OAuth2 Proxy for authentication. When true, Cryostat will authenticate users via OAuth2 Proxy.
+  enabled: true
+  ## @param authentication.oauth2Proxy.htpasswdFile Path to the htpasswd file for additional user authentication
+  htpasswdFile: "templates/auth_proxy_htpasswd"
+  ## @param authentication.oauth2Proxy.cookieSecret Secret key for encrypting OAuth2 Proxy cookies
+  cookieSecret: __24_BYTE_COOKIE_SECRET_
```

Comment:
This should allow the user to specify an override if they have some particular need, but if they do not then it should not default to a hardcoded value. Then it isn't a "secret" anymore :-) this should use a generated value like other secrets within `_helpers.tpl`.

Reply:
```
$ helm install cryostat ./charts/cryostat
Error: INSTALLATION FAILED: cannot load values.yaml: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{"include \"cryostat.defaultCookieSecret\" .":interface {}(nil)}
```

I think this can just be removed from `values.yaml`. If you look at how `cryostat.objectStorageSecretKey` and `-storage-secret-key` are used it should be a good example to follow for how this can be put together. `cryostat.databaseEncryptionKey` and `db-encryption-key` as well.

And please do try running `helm install` yourself after making changes so you can validate whether the changes are installable or produce the expected deployment outcome.